dipy/workflows/tests/test_workflow.py

```diff
@@ -0,0 +1,54 @@
+from nose.tools import assert_raises
+
+import os
+
+from nibabel.tmpdirs import TemporaryDirectory
+
+from dipy.data import get_data
+from dipy.workflows.segment import MedianOtsuFlow
+from dipy.workflows.workflow import Workflow
+
+
+def test_force_overwrite():
+    with TemporaryDirectory() as out_dir:
+        data_path, _, _ = get_data('small_25')
+        mo_flow = MedianOtsuFlow(output_strategy='absolute')
```

Comment:
if output_strategy is not 'append' or 'prepend' it is considered 'absolute'. This is bad design, I guess I should check for 'absolute' and raise an exception if is not. Or maybe I should use enums instead of dealing with strings?

What would you suggest?